src/ServiceControl.AcceptanceTests/Recoverability/MessageFailures/When_failed_message_searched_by_body_content.cs

```diff
@@ -0,0 +1,135 @@
+﻿namespace ServiceControl.AcceptanceTests.Recoverability.MessageFailures
+{
+    using System;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using CompositeViews.Messages;
+    using NServiceBus;
+    using NServiceBus.AcceptanceTesting;
+    using NServiceBus.AcceptanceTesting.Customization;
+    using NServiceBus.Settings;
+    using NUnit.Framework;
+    using TestSupport.EndpointTemplates;
+
+    class When_failed_message_searched_by_body_content : AcceptanceTest
+    {
+        [Test]
+        public async Task Should_be_found_when_fulltext_search_enabled()
+        {
+            // setting it even if it is the default
+            SetSettings = settings => settings.EnableFullTextSearchOnBodies = true;
+
+            var searchString = "42";
```

Comment:
In theory, it's possible that the string `42` may appear in a guid and therefore this test (and the one below) could randomly fail. Maybe changing it `forty-two` :)

Reply:
Addressed